BouncyCastle-JCA/src/CertPathTrustManagerParameters.crysl

```diff
@@ -4,10 +4,10 @@ OBJECTS
 	java.security.cert.CertPathParameters params;
 
 EVENTS
-	C: CertPathTrustManagerParameters(params);	
+	c: CertPathTrustManagerParameters(params);	
 	
 ORDER
-	C
+	c
```

Comment:
Can you check for consistency in the naming? You changed 'cP' to clear and often 'c1' to 'con'. If 'c' here is a constructor, shouldn't it be 'con'?